HighwayPatrol.h

```diff
@@ -1,11 +1,12 @@
 #pragma once 
 
 #include "Vehicle.h"
+#include "Highway.h"
 
 struct HighwayPatrol : Vehicle 
```

Comment:
Don't include what you don't use.
Forward-declare pointers and references that aren't **used** in header files.
Include in the cpp file.
- [ ] revise